aoe4/handlers.js

```diff
@@ -11,7 +11,7 @@ async function getPlayerWinRate(player, opponent, thresholdHours) {
   }
 
   // Fetch the first 50 games for all specified profileIds and merge them.
-  var games = (await Promise.all(playerProfileIds.map(p => aoe4.getPlayerGames(p)))).filter(v => v !== null).flat().sort((a, b) => b.game_id - a.game_id);
+  var games = (await Promise.all(playerProfileIds.map(p => aoe4.getPlayerGames(p)))).filter(v => v !== null).flat().filter(g => !!g.duration).sort((a, b) => b.game_id - a.game_id);
```

Comment:
I thought /games/last was only returning completed ones, but that was apparently an caching issue. I don't know what ppl want `!match` to return.

Reply:
I've only changed code for !winrate - But yes, that's a good question.
Maybe there should be a different command (or a query parameter option) to include or exclude ongoing games.
In any case, fixing !match is a bit harder since /games returns different objects than /games/last

Reply:
> I've only changed code for !winrate - But yes, that's a good question.

Ah yes, and for winrate the ongoing match doesn't matter. But you probably shouldn't filter it, it's still a valid game that should count toward the rolling window. Instead we should filter it from the statistic.
If this is the first game today, that'll result in 0 games (0-0) to be shown, instead of showing the previous day winrate.
So if a streamer just started his first match of the day, !winrate will properly reflect that.

Agreed on that !match is a bit more tricky. Let's leave that for another discussion.

Reply:
> Instead we should filter it from the statistic.
If this is the first game today, that'll result in 0 games (0-0) to be shown, instead of showing the previous day winrate.
A very good point, I hadn't considered that. I'll change it.